tests/memfile/run-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+for i in {1..64}; do
+	echo "00001111000000001111111100000000" >> tempfile1.dat
```

Comment:
Wouldn't this keep appending to `tempfile1.dat` on each run? Why not just commit this file?

Reply:
XD I deleted a previous rm of the file :S Ok, I will add tempfile1.dat but tempfile2.dat will be still a copy into temp, to simulate a file in another directory.

Reply:
FYI. Because is not more a `tempfile` I changed the name to content1.dat and content2.dat.